layouts/questions-index.html

```diff
@@ -0,0 +1,3 @@
+<%= yield %>
+<% questions = @items.find_all('/question/*.md').sort_by { |q| DateTime.parse(q[:creation_date]) }.reverse %>
+<%= render '/question-list.html', :questions => questions %>
```

Comment:
Using a layout as a partial like this is not my favorite. I think this will go away if we make the actual content file html.

Reply:
What do you mean by making the content file html?

Reply:
Sending a PR to your PR.

Reply:
https://github.com/ros-infrastructure/answers-archive/pull/3